src/transformers/trainer.py

```diff
@@ -1243,7 +1243,10 @@ def _rotate_checkpoints(self, use_mtime=False) -> None:
             shutil.rmtree(checkpoint)
 
     def evaluate(
-        self, eval_dataset: Optional[Dataset] = None, ignore_keys: Optional[List[str]] = None
+        self,
+        eval_dataset: Optional[Dataset] = None,
+        ignore_keys: Optional[List[str]] = None,
+        metric_key_prefix: Optional[str] = None,
```

Comment:
Thinking a bit more, why not just pass the default here (since we default to "eval" later anyway)? (Same in predict and prediction_loop)